CoreUtilities/src/au/gov/asd/tac/constellation/utilities/json/JsonUtilities.java

```diff
@@ -256,10 +290,10 @@ public Boolean next() {
      */
     public static String getTextValue(String attribute, JsonNode node) {
         if (node.has(attribute)) {
-            return node.get(attribute).textValue();
-        } else {
-            return null;
+            JsonNode attributeNode = node.get(attribute);
```

Comment:
final

Reply:
Have changed these and a few others in untouched code